react/src/components/forms/SelectBox/index.js

```diff
@@ -42,36 +43,42 @@ class SelectBox extends React.Component {
     }
     const selectClassNames = this.props.required ? 'ma__select-box__select js-dropdown-select js-required' : 'ma__select-box__select js-dropdown-select';
     const { selected } = this.state;
-    const { stackLabel } = this.props;
+    const {
+      label, id, options, stackLabel
+    } = this.props;
     const labelClassNames = stackLabel ? 'ma__select-box__label' : 'ma__label--inline ma__label--small';
     const selectBoxInline = stackLabel ? '' : 'ma__select-box__field--inline';
     const getValueByText = (array = [], text) => {
       const matchedItem = array.find((item) => item.text === text);
       const matchedValue = matchedItem && matchedItem.value;
       return matchedValue;
     };
-    const valueInOptions = getValueByText(this.props.options, selected);
-    const selectedValue = valueInOptions || this.props.options[0].value;
+    const valueInOptions = getValueByText(options, selected);
+    const selectedValue = valueInOptions || options[0].value;
```

Comment:
What happens when options[0] does not exist? Is options a required prop?

Reply:
yes, updated the prototype. I envision a lot of clean up work in form elements coming up. I didn't pay much attention to things other than the label in this pr